README.md

```diff
@@ -1,9 +1,10 @@
-mongoose_jid
-=====
+# mongoose_jid
 
-An OTP library
+[![Hex pm](http://img.shields.io/hexpm/v/mongoose_jid.svg?style=flat)](https://hex.pm/packages/mongoose_jid)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/mongoose_jid/)
+[![Downloads](https://img.shields.io/hexpm/dt/mongoose_jid.svg)](https://hex.pm/packages/mongoose_jid)
+[![License](https://img.shields.io/hexpm/l/mongoose_jid.svg)](https://github.com/esl/mongoose_jid/blob/master/LICENSE)
 
-Build
------
+An XMPP library for parsing JIDs, fully compliant with [RFC3920 Section 3.1](https://datatracker.ietf.org/doc/html/rfc3920#section-3.1)
```

Comment:
This RFC is obsolete. Do you know if it is compliant with the newer ones?

Reply:
Not at all, but that's kinda ok... The only difference though is the string normalisation algorithm. The old XMPP RFC uses stringprep profiles, while the new ones uses PRECIS, see https://datatracker.ietf.org/doc/html/rfc6122#section-1.1. The issue is, there's no PRECIS library in Erlang, at least none that I could find a few months ago when I tried looking into it, so until there's any, XMPP addresses are still using the old normalisation algorithms.

Reply:
Wait, I'm mixing RFC numbers. Nevertheless we don't have PRECIS, but I'll make sure to have the right RCF numbers.